code/modules/antagonists/changeling/changeling.dm

```diff
@@ -1049,8 +1048,8 @@
 /datum/outfit/changeling_space
 	name = "Changeling (Space)"
 
-	head = /obj/item/clothing/head/helmet/space/changeling
-	suit = /obj/item/clothing/suit/space/changeling
+	head = /obj/item/clothing/head/helmet/space
+	suit = /obj/item/clothing/suit/space
```

Comment:
that's pretty lame, why would an organic fleshy meteor spawn you with a space suit? should either be the chitin armor or just you being naked

Reply:
(or make the changeling spawn as a headslug but i think people would be upset)

Reply:
Sure I can make you nude instead